src/pypromice/process/value_clipping.py

```diff
@@ -29,31 +29,34 @@ def clip_values(
 
     variable_limits = var_configurations[cols].dropna(how="all")
     for var, row in variable_limits.iterrows():
-        if var not in list(ds.variables):
-            continue
-
-        if var in ["rh_u_cor", "rh_l_cor"]:
-            ds[var] = ds[var].where(ds[var] >= row.lo, other=0)
-            ds[var] = ds[var].where(ds[var] <= row.hi, other=100)
-
-            # Mask out invalid corrections based on uncorrected var
-            var_uncor = var.split("_cor")[0]
-            ds[var] = ds[var].where(~np.isnan(ds[var_uncor]), other=np.nan)
-
-        else:
-            if ~np.isnan(row.lo):
-                ds[var] = ds[var].where(ds[var] >= row.lo)
-            if ~np.isnan(row.hi):
-                ds[var] = ds[var].where(ds[var] <= row.hi)
-
-        other_vars = row.OOL
-        if isinstance(other_vars, str) and ~ds[var].isnull().all():
-            for o in other_vars.split():
-                if o not in list(ds.variables):
-                    continue
-                else:
-                    if ~np.isnan(row.lo):
-                        ds[var] = ds[var].where(ds[var] >= row.lo)
-                    if ~np.isnan(row.hi):
-                        ds[var] = ds[var].where(ds[var] <= row.hi)
+        try:
+            if var not in list(ds.variables):
+                continue
+    
+            if var in ["rh_u_cor", "rh_l_cor"]:
+                ds[var] = ds[var].where(ds[var] >= row.lo, other=0)
+                ds[var] = ds[var].where(ds[var] <= row.hi, other=100)
+    
+                # Mask out invalid corrections based on uncorrected var
+                var_uncor = var.split("_cor")[0]
+                ds[var] = ds[var].where(~np.isnan(ds[var_uncor]), other=np.nan)
+    
+            else:
+                if ~np.isnan(row.lo):
+                    ds[var] = ds[var].where(ds[var] >= row.lo)
+                if ~np.isnan(row.hi):
+                    ds[var] = ds[var].where(ds[var] <= row.hi)
+    
+            other_vars = row.OOL
+            if isinstance(other_vars, str) and ~ds[var].isnull().all():
+                for o in other_vars.split():
+                    if o not in list(ds.variables):
+                        continue
+                    else:
+                        if ~np.isnan(row.lo):
+                            ds[var] = ds[var].where(ds[var] >= row.lo)
+                        if ~np.isnan(row.hi):
+                            ds[var] = ds[var].where(ds[var] <= row.hi)
```

Comment:
I don't see any effects from the other vars loop. The `o` variable is only used in line 53.
The lines 56-59 are identical with 45-48.

Reply:
Hm. I thought this was quite an important step as it is looking for variables that inherently rely on other variables (defined in the `ool` column of the `variables.csv` file. But if it is not doing anything to any of our data then we could look at changing this in a separate PR.